build-source-tarball.sh

```diff
@@ -148,9 +148,6 @@ git submodule foreach --quiet --recursive '
 # Now re-uberclone into the tarball src directory.  Since we reuse the .gitdirs, this shouldn't hit the network at all.
 ignored_repos="https://dev.azure.com/dnceng/internal/_git/dotnet-optimization;https://dev.azure.com/devdiv/DevDiv/_git/DotNet-Trusted;https://devdiv.visualstudio.com/DevDiv/_git/DotNet-Trusted;https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-optimization;https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup;https://github.com/dotnet/source-build-reference-packages"
 
-#export the LC_LIB_PATH for libgit2 so file as fedora fails to find it in the repodir
-export LD_LIBRARY_PATH=$CLI_PATH/tools/.store/microsoft.dotnet.darc/$DarcVersion/microsoft.dotnet.darc/$DarcVersion/tools/netcoreapp2.1/any/runtimes/rhel-x64/native/
```

Comment:
We don't need this now? Is this due to netcoreapp3.0 update?

Reply:
I believe it's actually fixed by https://github.com/libgit2/libgit2sharp/pull/1714, but that change relied on some 3.0 runtime changes, so the fix didn't actually work until we switched to 3.0.

Reply:
Thanks for the explanation. 👍